internal/commands/test.go

```diff
@@ -185,6 +188,7 @@ func NewTestCommand(ctx context.Context) *cobra.Command {
 	cmd.Flags().StringSliceP("data", "d", []string{}, "A list of paths from which data for the rego policies will be recursively loaded")
 
 	cmd.Flags().StringSlice("proto-file-dirs", []string{}, "A list of directories containing Protocol Buffer definitions")
+	cmd.Flags().BoolP("tls", "s", true, "Use TLS to access the registry")
```

Comment:
Please make this a `Bool()` call rather than `BoolP`. A single-letter alias of `s` for the `tls` flag does not make sense.